### ISSUE TITLE
[Docs] pypi project description does not render correctly since 61.0.0

### ISSUE THREAD

**Issue by ShaneHarvey** [pypa/setuptools#4008]
### Summary

setuptools' pypi project description does not render correctly since 61.0.0. Compare:
- https://pypi.org/project/setuptools/60.10.0/
- https://pypi.org/project/setuptools/61.0.0/

### Additional Information

Fixing this makes setuptools' pypi project description more readable.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by abravalheri** [comment]
Hi @ShaneHarvey thabk you for opening the issue.

That is a bit weird...

https://web.archive.org/web/20230329061036/https://pypi.org/project/setuptools/61.0.0/ seems to indicate the project page working correct for v61in 29/Mar/2023.

The artifacts in PyPI are immutable,  so I wonder if PyPI changed some algorithm recently...

**Comment by abravalheri** [comment]
Probably caused by https://github.com/pypi/warehouse/issues/14064.
I am contacting the PyPI team to see what is the best action.

**Comment by di** [comment]
This was due to https://github.com/pypi/warehouse/issues/14064. We haven't mass-re-rendered descriptions that failed to render as a result of that bug yet.

I went ahead and manually re-rendered all the `setuptools` descriptions and purged the old ones from our cache.

> The artifacts in PyPI are immutable, so I wonder if PyPI changed some algorithm recently...

While this is true, the method by which we render this immutable metadata does change as new versions of our dependencies for rendering are released, and when that happens we go back and re-render all "old" descriptions.
